framework/hpp_api_vulkan_sample.cpp

```diff
@@ -485,6 +485,15 @@ void HPPApiVulkanSample::submit_frame()
 			present_info.setWaitSemaphores(semaphores.render_complete);
 		}
 
+		vk::DisplayPresentInfoKHR disp_present_info{};
```

Comment:
You don't need to initialize the vk-structs with `{}`, as they have a default constructor.

Reply:
Done